src/app/issues-viewer/issues-viewer.component.ts

```diff
@@ -101,7 +96,7 @@ export class IssuesViewerComponent implements OnInit, AfterViewInit, OnDestroy {
     this.githubService.getUsersAssignable().subscribe((x) => (this.assignees = x));
 
     // Fetch issues
-    this.issueService.reloadAllIssues();
+    this.issueService.reloadAllIssues().pipe(catchError((err) => 'Failed to fetch issues for repo. Check your repo name.'));
```

Comment:
I don't think this line would do anything.
Maybe you intended to call the logger in the callback function?

Reply:
thank you for catching this, this was to be deleted :)